### DISCUSSION TITLE
'paths-ignore' doesn't work for the path '.github/project.yaml

### DISCUSSION THREAD

**Discussion by rishabprasad005** [community/community#26477] [Actions]
<p>Below is the workflow trigger I am using:-</p>
<pre><code>on:

  push:

    paths-ignore:

      - '.github/project.yaml'

      - 'README.md'

      - LICENSE

    branches:

      - master
</code></pre>
<p>   </p>
<p>paths-ignore works for me and the workflow doesn’t gets triggered if I do any changes on the ‘README.md’ and ‘LICENSE’ file. However, it doesn’t work for the path ‘.github/project.yaml’. I want that the workflow should not get triggerred if the changes are made only on the ‘project.yaml’ file which is present at location ‘.github/project.yaml’. however, it gets triggered currently and ‘paths-ignore’ doesn’t seems to be working for me in case of ‘.github/project.yaml’</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/rishabprasad005">@rishabprasad005</a> ,</p>
<p>Your code ignore setting works well for ‘.github/project.yaml’ file on my side.</p>
<p>Could you please confirm the project.yaml is in the ‘.github’ folder not the ‘.github/workflow’ folder?</p>
<p>And if there any typo possible?</p>
<p>If all looks fine but setting still not work, could you please share your repo for a check? Thanks.</p>

**Comment by rishabprasad005** [comment]
<p>Hi <a class="mention" href="/weide-zhou">@weide-zhou</a> ,</p>
<p>I am really sorry for the trouble. turns out I had  typo. Thanks for responding.</p>
